server/fakedb/Schedule.csv

```diff
@@ -0,0 +1,22 @@
+﻿eventId,start,end,locationLatitude,locationLongitude,repeat
+1,2019-12-26,2019-12-28,36.675439,140.6715934,
```

Comment:
No hour:minute? How would people know when to attend the meeting?

Reply:
I was referring to the examples :) I think it'll more accurate to test features if we have realistic data

Reply:
I added time info on start/end mock data.